core/src/main/scala/org/apache/spark/internal/config/Deploy.scala

```diff
@@ -19,48 +19,59 @@ package org.apache.spark.internal.config
 
 private[spark] object Deploy {
   val RECOVERY_MODE = ConfigBuilder("spark.deploy.recoveryMode")
+    .version("0.8.1")
```

Comment:
What is 0.8.1 here?

Reply:
The earliest version of the configuration item.

Reply:
Ideally it might be better to leave a JIRA ID as a comment on each line.

Reply:
Many early commits did not have a jira ID.

Reply:
You can show the commit log too. That should deduplicate the review overhead because probably reviewers should do exactly the same thing as you did to verify :-).

Reply:
OK. But commit log is too long. Can I just add the URL corresponding to the commit?

Reply:
yeah just commit url or hash anything is fine.

Reply:
OK